Makefile

```
@@ -349,3 +349,27 @@ integration-test:
 	else\
 		$(GINKGO) -v $(TEST_ARGS) --fail-fast --focus-file=$(TEST_FILE) test/integration;\
 	fi
+
+#hosted
+
+.PHONY: bootstrap-hosted
+bootstrap-hosted: clone-hosted install-hosted copy-config install-crds install-resources kind-deploy-policy-framework kind-deploy-policy-controllers
+
+.PHONY: install-hosted
+install-hosted: 
+	@cd ./governance-policy-addon-controller && KIND_VERSION=$(KIND_VERSION) HOSTED_MODE=true ./build/manage-clusters.sh
+
+.PHONY: 
+copy-config:
+	@cp ./governance-policy-addon-controller/policy-addon-ctrl1.kubeconfig ./kubeconfig_hub
+	@cp ./governance-policy-addon-controller/policy-addon-ctrl1.kubeconfig-internal ./kubeconfig_hub_internal
+	@cp ./governance-policy-addon-controller/policy-addon-ctrl2.kubeconfig ./kubeconfig_managed
+	@cp ./governance-policy-addon-controller/policy-addon-ctrl2.kubeconfig-internal ./kubeconfig_managed_internal
```

I think these targets should be combined since the goal is to deploy Kind clusters for the framework tests, I don't think there's a need to run them separately.
```suggestion
.PHONY: kind-install-hosted
kind-install-hosted: 
	@cd ./governance-policy-addon-controller && KIND_VERSION=$(KIND_VERSION) HOSTED_MODE=true ./build/manage-clusters.sh
	@cp ./governance-policy-addon-controller/policy-addon-ctrl1.kubeconfig ./kubeconfig_hub
	@cp ./governance-policy-addon-controller/policy-addon-ctrl1.kubeconfig-internal ./kubeconfig_hub_internal
	@cp ./governance-policy-addon-controller/policy-addon-ctrl2.kubeconfig ./kubeconfig_managed
	@cp ./governance-policy-addon-controller/policy-addon-ctrl2.kubeconfig-internal ./kubeconfig_managed_internal
```